perun.go

```diff
@@ -228,6 +228,98 @@ type NodeConfig struct {
 
 }
 
+// APIErrorV2 represents the newer version of error returned by node, session
+// and channel APIs.
+//
+// Along with the error message, this error type assignes to each error an
```

Comment:
```suggestion
// Along with the error message, this error type assigns to each error an
```

Reply:
Updated in 47c015884abb63d42121a1c227363cd3db16f387.